application/src/main/java/org/togetherjava/tjbot/commands/free/FreeCommand.java

```diff
@@ -199,17 +199,13 @@ private boolean handleShouldBeProcessed(@NotNull final SlashCommandEvent event)
      * @param channel the text channel the status message will be posted in.
      */
 
-    /*
-     * event.replyEmbeds(new EmbedBuilder() .setDescription(tagSystem.getTag(id).orElseThrow()
-     * .setFooter(event.getUser() + " - used /tag foo") .setColor(TagSystem.AMBIENT_COLOR)
-     * ).queue();
-     */
     public void displayStatus(@NotNull TextChannel channel) {
         final Guild guild = channel.getGuild();
 
         String messageTxt = buildStatusMessage(guild);
         MessageEmbed embed = new EmbedBuilder().setTitle(STATUS_TITLE)
             .setDescription(messageTxt)
+            .setFooter(channel.getJDA().getSelfUser().toString())
```

Comment:
Dont toString the user, you wont get what you expect. Do what the code did in the past, namely getting the name of the user.